accounts/views.py

```diff
@@ -1069,3 +1065,33 @@ def pending(request):
     }
     tvars.update(paginate(request, pendings, settings.SOUNDS_PENDING_MODERATION_PER_PAGE))
     return render(request, 'accounts/pending.html', tvars)
+
+
+@csrf_exempt
+def donation_complete(request):
+    params = {'cmd': '_notify-validate'}
+    for key, value in request.POST.items():
+        params[key] =  value
```

Comment:
This view listens to a notification made from paypal when someone makes a donation, here we re-send all the data to paypal for validating it. 
There are around 30 parameters and I don't this is an issue.